src/ErrorsStack.ts

```diff
@@ -0,0 +1,28 @@
+interface Params {
+  name: string
+  message: string
```

Comment:
```suggestion
  message?: string
```

Não deveria ser assim?

Reply:
Esse `message` vai ser usado como default, pra caso não seja passado nas construções seguintes, ai precisa ser passado mesmo